packages/orbit-components/src/Tabs/components/TabList/index.tsx

```diff
@@ -29,14 +16,42 @@ const TabList = ({
   dataTest,
   fullWidth,
 }: Props) => {
+  const cssVars = {
+    "--tab-list-padding": typeof padding === "string" ? padding : undefined,
+    "--tab-list-margin": typeof margin === "string" ? margin : undefined,
+    "--tab-list-padding-top": typeof padding === "object" ? padding.top : undefined,
+    "--tab-list-padding-right": typeof padding === "object" ? padding.right : undefined,
+    "--tab-list-padding-bottom": typeof padding === "object" ? padding.bottom : undefined,
+    "--tab-list-padding-left": typeof padding === "object" ? padding.left : undefined,
+    "--tab-list-margin-top": typeof margin === "object" ? margin.top : undefined,
+    "--tab-list-margin-right": typeof margin === "object" ? margin.right : undefined,
+    "--tab-list-margin-bottom": typeof margin === "object" ? margin.bottom : undefined,
+    "--tab-list-margin-left": typeof margin === "object" ? margin.left : undefined,
+  } as React.CSSProperties;
+
   return (
-    <StyledTabListWrapper
+    <div
+      className={cx(
+        fullWidth ? "w-full" : "w-auto",
+        cssVars["--tab-list-padding"] != null && "p-[var(--tab-list-padding)]",
+        cssVars["--tab-list-margin"] != null && "m-[var(--tab-list-margin)]",
+        cssVars["--tab-list-padding"] != null && [
```

Comment:
this is actually false. if `--tab-list-padding` is not `undefined`, then `padding` is a `string`. in that case, the below vars (`...padding-top`, etc.) will be `undefined`

you need to check either if `padding` is an object, or preferably, if e.g. `padding-top` is not `undefined`